filecoin-proofs/tests/pieces.rs

```diff
@@ -113,30 +113,38 @@ fn test_verify_simple_pieces() {
     // / \  / \
     // a  b c  d
 
-    let (a, b, c, d): ([u8; 32], [u8; 32], [u8; 32], [u8; 32]) = rng.gen();
```

Comment:
I am okay with this, but in general we should just put a clippy `allow` on this function